app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/viewmodel/OutputViewModel.kt

```diff
@@ -39,35 +39,23 @@ class OutputViewModel : ViewModel() {
     var output: List<String> by mutableStateOf(listOf())
         private set
 
-    fun appendOutput(text: String) {
+    fun appendOutput(text: String, withTimestamp: Boolean = true) {
```

Comment:
I decided against "hasTimestamp" because it did not really match with the "appendOutput"-method name. In this context it might suggest, that the text parameter "has a timestamp", that's why I decided to go for the approach of naming it "appendOutput withTimestamp true". 

I now changed it and replaced the text by a "LogEntry". A LogEntry is basically a list of strings, which belong to the same "event" and therefore only contain one timestamp.